app/views/contact/new.html.slim

```diff
@@ -0,0 +1,34 @@
+- title t('titles.contact')
+
+h1.heading = t('headings.contact')
+
+= simple_form_for(@contact_form, url: contact_path, wrapper: :base,
+    html: { autocomplete: 'off', method: :post, role: 'form' }) do |f|
+  = f.error_notification
+  h4.mb-tiny = t('contact.intro')
+  hr.mt0.mb-12p
+  .mb4
+    .mb-tiny
+      = f.label :want_learn, class: 'checkbox'
+        = f.input :want_learn, as: :boolean, wrapper: false
```

Comment:
i had to add this explicitly because the input needs to nested within the label for this checkbox; (simple form defaults to adding them on the same DOM level)